types/address.go

```diff
@@ -70,10 +71,15 @@ const (
 
 // Address is a common interface for different types of addresses used by the SDK
 type Address interface {
-	Equals(Address) bool
+	proto.Sizer
+	proto.Marshaler
+
+	Equal(Address) bool
```

Comment:
Renamed Equals to Equal, so that it fits protobuf custom type's interface https://github.com/gogo/protobuf/blob/master/custom_types.md#custom-type-method-signatures